index.html

```diff
@@ -26,24 +26,181 @@
 
   <section class="container">
        
-    <div class=" box name"><h3>Samuel Ackon</h3></div>
-    <div class=" box intro"><h4>Hello! I am a software developer! 
+    <div class=" box name"><h1>Samuel Ackon</h1></div>
+    <div class=" box intro"><p>Hello! I am a software developer! 
         I can help you build a product, feature or website. 
         Taka a look of my works. 
         If you like what you see and have a 
         project you need coded,
-         don’t hesitate and contact me</h4></div>
+         don’t hesitate and contact me</p></div>
        <div class="rectangle">
           <div>Start collaboration</div></div>
 
        <div class="designs">
           <div class="image3"><img src="assets/image_geometry_2.png" alt="#" class="shape002"></div>
       </div>
- 
+  </section>
+</div>
+
+
+<div class="Hallelujah">
+    
+    <section class="Works">
+        <div class="Project-name"><h2>Projects</h2></div>
+        <div class="work one">
+            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
+            <div class="project summary">
+                <div class="nameP"><p>Project name goes here</p></div>
+                <div class="technology">
+                    <div class="tech1"><p>HTML/CSS</p></div>
+                    <div class="tech1"><p>Ruby on Rails</p></div>
+                    <div class="tech1"><p>JavaScript</p></div>
+                </div>
+                <div class="project-view">
+                    <div class="word"><p>See this project</p></div>
+                    <div class="union"><img src="assets/work/Union.png" class="arrow" alt="#"></div>
```

Comment:
- [ ] Please, consider that the button (e.g., "See this project") is a link `a` or `button` tag